registry/constants.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class OrganizationStatus(Enum):
+    DRAFT = "DRAFT"
+    APPROVAL_PENDING = "APPROVAL_PENDING"
+    APPROVED = "APPROVED"
```

Comment:
should this be in the domain module as they are concepts related to the domain ?